notebooks/official/migration/sdk-hyperparameter-tuning.ipynb

```diff
@@ -587,10 +503,10 @@
       "source": [
         "#### Task.py contents\n",
         "\n",
-        "In the next cell, you write the contents of the hyperparameter tuning script task.py. I won't go into detail, it's just there for you to browse. In summary:\n",
+        "In the next cell, you write the contents of the hyperparameter tuning script task.py. you won't go into detail, it's just there for you to browse. In summary:\n",
```

Comment:
- Change: "you won't go into detail, it's just there for you to browse."
- To: "It's there for you to browse."